course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -67,9 +67,56 @@
                     else if (data.results.length === 1) {
                         // TODO - POST user, handle results
                     }
+
+                    $scope.searchResults = $scope.filterResults(data.results);
                 })
                 .error($scope.ajaxErrorHandler);
         };
+        /*
+            concat the active flag with the prime_role_indicator
+            value to be able to sort records base on these values
+            active = (0 | 1) a value of 1 here trumps the prime_role_indicator
+            prime_role_indicator = ( "Y" | "N" | "")
+            1 > 0  true
+            '1:string' > '0:string' true
+            'Y' > 'N' true
+            This should let any records with a 1 in the active column float to the top
+            if there are non, records with a Y in the prime_role_indicator will float up
+            and records with both will float above each of those.
+         */
+        $scope.compare = function(a, b) {
```

Comment:
I'm usually not in favor of things that are visually less readable, but this is nice!! :)